backend/danswer/db/auth.py

```diff
@@ -35,12 +36,14 @@ def get_default_admin_user_emails() -> list[str]:
     return get_default_admin_user_emails_fn()
 
 
-def get_total_users(db_session: Session) -> int:
+def get_total_users_count(db_session: Session) -> int:
     """
     Returns the total number of users in the system.
     This is the sum of users and invited users.
     """
-    user_count = db_session.query(User).count()
+    user_count = (
```

Comment:
I feel like this is very slow. requires loading the email and running a python function on every single one.